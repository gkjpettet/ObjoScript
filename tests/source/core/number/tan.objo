System.print(0.tan) # Expect 0
System.print(1.tan) # Expect 1.557407725
System.print(0.5.tan) # Expect 0.54630249