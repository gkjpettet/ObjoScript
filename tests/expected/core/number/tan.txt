0
1.557407725
0.54630249